lib/newman_conway.rb

```diff
@@ -1,7 +1,26 @@
+# Time complexity: O(n) where n is the input value.
+# Space Complexity: O(n) where n is the input value.
+def newman_conway(num)
+  output_array = []
+  if num == 0
+    raise ArgumentError
+  elsif num == 1
+    return "1"
+  elsif num == 2
+    return "1 1"
```

Comment:
Do you really need these?  If you have the array, do you need these `elsif` conditions?